src/execution_plan/ops/op.h

```diff
@@ -42,6 +42,7 @@ typedef enum {
     OPType_NODE_BY_ID_SEEK = (1<<21),
     OPType_PROC_CALL = (1<<22),
     OPType_CONDITIONAL_VAR_LEN_TRAVERSE_EXPAND_INTO = (1<<23),
+    OPType_VALUE_HASH_JOIN = (1<<24),
```

Comment:
So many operations! We won't be able to keep this as a bitmask for too much longer.

Reply:
Agree, we can switch to enum